modules/standard/IO.chpl

```diff
@@ -3623,7 +3658,10 @@ private inline proc _write_binary_internal(_channel_internal:qio_channel_ptr_t,
 
 // Channel must be locked, must be running on this.home
 // x is ref (vs out) because it might contain a literal string.
-private inline proc _read_one_internal(_channel_internal:qio_channel_ptr_t, param kind:iokind, ref x:?t):syserr where _isIoPrimitiveTypeOrNewline(t) {
+private inline proc _read_one_internal(_channel_internal:qio_channel_ptr_t,
+                                       param kind:iokind,
+                                       ref x:?t,
+                                       loc:locale):syserr where _isIoPrimitiveTypeOrNewline(t) {
```

Comment:
It is intentional. These functions are overloaded - have two implementations, one for built-in types and another for anything else. I needed the loc:locale argument on the "anything else" version so changed both overloads.